app/assets/javascripts/app/global.js

```diff
@@ -0,0 +1,213 @@
+	      e.preventDefault();
+	      e.stopPropagation();
+
+	      // Remove the file preview.
+	      _this.removeFile(file);
+	      // If you want to the delete the file on the server as well,
+	      // you can do the AJAX request here.
+	    });
+
+	    // Add the button to the file preview element.
+	    file.previewElement.appendChild(removeButton);
+	  });
+	}
+};
+
+/**************************
+*
+*	EXT - for the quotes
+*
+**************************/
+(function($){
+	$.fn.extend({ 
+        //plugin name - rotaterator
+        rotaterator: function(options) {
+
+        	var defaults = {
+        		fadeSpeed: 600,
+        		pauseSpeed: 100,
+        		child:null
+        	};
+
+        	var options = $.extend(defaults, options);
+
+        	return this.each(function() {
+        		var o =options;
+        		var obj = $(this);                
+        		var items = $(obj.children(), obj);
+        		items.each(function() {$(this).hide();})
+        		if(!o.child){var next = $(obj).children(':first');
+        	}else{var next = o.child;
+        	}
+        	$(next).fadeIn(o.fadeSpeed, function() {
+        		$(next).delay(o.pauseSpeed).fadeOut(o.fadeSpeed, function() {
+        			var next = $(this).next();
+        			if (next.length == 0){
+        				next = $(obj).children(':first');
+        			}
+        			$(obj).rotaterator({child : next, fadeSpeed : o.fadeSpeed, pauseSpeed : o.pauseSpeed});
+        		})
+        	});
+        });
+        }
+    });
+})(jQuery);
+
+$(function(){
+
+	//home page quotes
+	$('#quotes').rotaterator({fadeSpeed:1000, pauseSpeed:3000});
+
+	//always activate first tab
+	$('#myTab a:eq(1)').tab('show');
+
+    
+    //listings show more
+    $( "#more_make" ).bind( "click", function() {
+        $('#more_make_list').show();
+        $('#more_make_link').hide();
+        return false;
+    });    
+    $( "#less_make" ).bind( "click", function() {
+        $('#more_make_list').hide();
+        $('#more_make_link').show();
+        return false;
+    });
+	//the graph
+    if($("#visualization").length > 0){
+
+
+          var css_id = "#visualization";
+        var data = [
+            {label: 'BMW 525D SE TOURING SILVER', data: [[1,700], [2,600], [3,400], [4,390], [5,300], [6,300], [7,300]]},
+            {label: 'SEAT Leon 1.6 TDI CR S 5dr (2010)', data: [[1,800], [2,600], [3,400], [4,400], [5,500], [6,400], [7,400]]},
+            {label: '2008 Scuderia Spider 16M', data: [[1,800], [2,200], [3,200], [4,200], [5,100], [6,50], [7,0]]}
+        ];
+        var options = {
+            series: {stack: 0,
+                     lines: {show: true, steps: false },
+                     bars: {show: false, barWidth: 0.9, align: 'center'}
+            },
+            xaxis: {ticks: [[1,'Sun'], [2,'Mon'], [3,'Tue'], [4,'Wed'], [5,'Thu'], [6,'Fri'], [7,'Sat']]}
+        };
+
+        $.plot($(css_id), data, options);
+    }
+
+
+	//make text area bigger
+	var textarea_height = $('textarea.expand').height();

```

Comment:
Identifier 'textarea_height' is not in camel case.<br>Mixed double and single quotes.